### DISCUSSION TITLE
Push to branch not triggering workflow

### DISCUSSION THREAD

**Discussion by vvasuki** [community/community#26433] [Actions]
<p>Hi!</p>
<p>Consider this workflow: <a href="https://github.com/sanskrit/ashtadhyayi_com_transforms/blob/b45a9d5295138e4cb99083f715701be2b420b46e/.github/workflows/commentary_separation.yml" rel="noopener nofollow ugc">https://github.com/sanskrit/ashtadhyayi_com_transforms/blob/b45a9d5295138e4cb99083f715701be2b420b46e/.github/workflows/commentary_separation.yml</a> which ought to be triggered by pushes to the trigger_branch. But that is just not happening (screenshots below).</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/1/21ae35786da95c2adc5d441ca246589a2c4e51ea.png" data-download-href="/uploads/short-url/4NWZPDGVsVglln83Bq5JHoMaM9c.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2664797/181097637-6a29e4be-01be-4a40-9ade-ab7f74d391b4.png" alt="image" data-base62-sha1="4NWZPDGVsVglln83Bq5JHoMaM9c" width="690" height="327" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21ae35786da95c2adc5d441ca246589a2c4e51ea_2_690x327.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21ae35786da95c2adc5d441ca246589a2c4e51ea_2_1035x490.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21ae35786da95c2adc5d441ca246589a2c4e51ea_2_1380x654.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/1/21ae35786da95c2adc5d441ca246589a2c4e51ea_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1380×655 54.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/6/762933db2f079b901b3e546992028d4603b5744a.png" data-download-href="/uploads/short-url/gRiCi2Nllleh9BVrrFJ1NV6AaKu.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/2664797/181097638-d1d2d035-d2f4-4e16-84cc-9812d98e64f3.png" alt="image" data-base62-sha1="gRiCi2Nllleh9BVrrFJ1NV6AaKu" width="690" height="291" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/762933db2f079b901b3e546992028d4603b5744a_2_690x291.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/6/762933db2f079b901b3e546992028d4603b5744a_2_1035x436.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/6/762933db2f079b901b3e546992028d4603b5744a_2_1380x582.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/762933db2f079b901b3e546992028d4603b5744a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1866×787 76 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>What’s wrong?</p>

**Comment by Simran-B** [comment, marked as answer]
<blockquote>
<p>[…] if the event occurred on a particular repository branch, then the workflow files must be present in the repository on that branch.</p>
</blockquote>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration" target="_blank" rel="noopener nofollow ugc">Events that trigger workflows - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Your workflow is not present in the <code>trigger_branch</code>:</p>
<p><a href="https://github.com/sanskrit/ashtadhyayi_com_transforms/tree/trigger_branch/.github" rel="noopener nofollow ugc">https://github.com/sanskrit/ashtadhyayi_com_transforms/tree/trigger_branch/.github</a></p>
